js/apps/admin-ui/src/identity-providers/add/GeneralSettings.tsx

```diff
@@ -1,19 +1,31 @@
 import { RedirectUrl } from "../component/RedirectUrl";
 import { ClientIdSecret } from "../component/ClientIdSecret";
+import { TextField } from "../component/TextField";
 import { DisplayOrder } from "../component/DisplayOrder";
 
+
 type GeneralSettingsProps = {
   id: string;
+  showclientidsecret?: boolean;
```

Comment:
As mentioned before we prefer to use camel case here.
```suggestion
  showClientIdSecret?: boolean;
```